googler

```diff
@@ -1175,13 +1206,22 @@ class GoogleParser(html.parser.HTMLParser):
             self.set_handlers_to('result')
             return 'result'
 
+        # Autocorrect
+        if 'spell_orig' in self.classes(attrs) and attrs.get('id') not in {'sifm', 'sifl'}:
```

Comment:
I'll throw a span tag here. In the beginning it was an a tag, just like `a.spell`, but that doesn't work with `--noua`.